taskqueue/gcloud/aio/taskqueue/taskqueue.py

```diff
@@ -228,21 +229,22 @@ def __init__(self, project, service_file, task_queue, session=None,
 
 
 class LocalTaskQueue(object):
-
     """
     An asynchronous in-memory Task Queue
     """
-
+    # pylint: disable=too-many-instance-attributes
     def __init__(self, *args, **kwargs):  # pylint: disable=unused-argument
-
         self.queue = asyncio.Queue()
+        self.deleted = {}
         self.leased = {}
         self.ready = {}
-        self.deleted = {}
 
-        self.task_queue = kwargs.get('task_queue') or 'q:{}'.format(
-            self.next_id()
-        )
+        self.duration = 0
+        self.start_time = None
```

Comment:
`start_time` is the time the task queue first leased a task and `duration` is the length of time the queue has been operating with continuous tasks (i.e. not empty)? Is this for analytics (ex. recording utilization)?

Reply:
Yes, indeed -- we're using this in a few internal projects to measure performance of processing _n_ tasks off the queue.